### PR TITLE
Migrate Critical Constants and Acentric Factors

### PR DESCRIPTION
**Pull request by saikat-samanta**
Migrate Critical Constants and Acentric Factors

### CODE REVIEW

File: src/data/critical-constants-acentric-factor.ts [hunk 1]
```diff
@@ -15,4 +15,268 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    criticalTemperature: 606,
+    criticalPressure: 4,
+    criticalVolume: 0.304,
+    criticalCompressibilityFactor: 0.241,
+    acentricFactor: 0.455328,
+  },
+  Acetone: {
+    name: 'Acetone',
+    formula: 'C3H6O',
+    CAS: ' 67-64-1',
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:54]:
```suggestion
    CAS: '67-64-1',
```

File: src/data/critical-constants-acentric-factor.ts [hunk 2]
```diff
@@ -15,4 +15,268 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    criticalTemperature: 508.2,
+    criticalPressure: 4.701,
+    criticalVolume: 0.209,
+    criticalCompressibilityFactor: 0.233,
+    acentricFactor: 0.306527,
+  },
+  Acetonitrile: {
+    name: 'Acetonitrile',
+    formula: 'C2H3N',
+    CAS: ' 75-05-8',
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:65]:
```suggestion
    CAS: '75-05-8',
```

File: src/data/critical-constants-acentric-factor.ts [hunk 3]
```diff
@@ -15,4 +15,268 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    criticalTemperature: 308.3,
+    criticalPressure: 6.138,
+    criticalVolume: 0.112,
+    criticalCompressibilityFactor: 0.268,
+    acentricFactor: 0.191185,
+  },
+  Acrolein: {
+    name: 'Acrolein',
+    formula: 'C3H4O',
+    CAS: ' 107-02-8',
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:87]:
```suggestion
    CAS: '107-02-8',
```

and so on.

**Review by qur786 — COMMENTED** [review]
All data is correct.

**Review by maneetgoyal — CHANGES_REQUESTED** [review]
White space issues but data is good

File: src/data/critical-constants-acentric-factor.ts [hunk 4]
```diff
@@ -15,4 +15,268 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    name: 'Acetic acid',
+    formula: 'C2H4O2',
+    CAS: '64-19-7',
+    molecularWeight: 60.052,
+    criticalTemperature: 591.95,
+    criticalPressure: 5.786,
+    criticalVolume: 0.177,
+    criticalCompressibilityFactor: 0.208,
+    acentricFactor: 0.466521,
+  },
```
Comment (sumitnegi7) [inline src/data/critical-constants-acentric-factor.ts:39]:
```suggestion
  },
  
```
@saikat-samanta  let's add a new line after  compound object data is finished

**Comment by saikat-samanta** [comment]
@manufac-analytics/devs, changes done 👍

**Review by maneetgoyal — APPROVED** [review]

**Review by sumitnegi7 — APPROVED** [review]

**Comment by sumitnegi7** [comment]
LGTM :+1:

**Pull request merged**